bangc-ops/mlu_op.h

```diff
@@ -5821,6 +5821,113 @@ mluOpFill_v3(mluOpHandle_t handle,
+ * @param[in] pts_desc
+ * The descriptor of pts, which contains the dimension and layout of the pts tensor.
+ * @param[in] pts
+ * Pointer to the MLU memory that stores the pts tensor.
+ * @param[in] pts_feature_desc
+ * The descriptor of pts_feature, which contains the dimension and layout of the pts_feature tensor.
+ * @param[in] pts_feature
+ * Pointer to the MLU memory that stores the pts_feature tensor.
+ * @param[in] workspace
+ * Pointer to the MLU memory that is used as an extra workspace for the
+ * ::mluOpMoeDispatchBackwardData operation.
+ * @param[in] workspace_size
+ * The size of the extra workspace in bytes that needs to be used in
+ * the ::mluOpMoeDispatchBackwardData operation. You can get the size of the workspace with
+ * the ::mluOpMoeDispatchBackwardDataWorkspaceSize function.
+ * @param[in] max_pts_each_voxel
+ * The maximum number of points per each voxel. An integer value which is the dimension of the pts_idx_of_voxels.
+ * @param[in] out_x
+ * An integer value which is the dimension of the pooled_features.
+ * @param[in] out_y
+ * An integer value which is the dimension of the pooled_features.
+ * @param[in] out_z
+ * An integer value which is the dimension of the pooled_features.
+ * @param[in] argmax_desc
+ * The descriptor of argmax, which contains the dimension and layout of the argmax tensor.
+ * @param[out] argmax
+ * Pointer to the MLU memory that stores the argmax tensor.
+ * @param[in] pts_idx_of_voxels_desc
+ * The descriptor of pts_idx_of_voxels, which contains the dimension and layout of the pts_idx_of_voxels tensor.
+ * @param[out] pts_idx_of_voxels
+ * Pointer to the MLU memory that stores the pts_idx_of_voxels tensor.
+ * @param[in] pooled_features_desc
+ * The descriptor of pooled_features, which contains the dimension and layout of the pooled_features tensor.
+ * @param[out] pooled_features
+ * Pointer to the MLU memory that stores the pooled_features tensor.
+ *
+ * @par Return
+ * - ::MLUOP_STATUS_SUCCESS, ::MLUOP_STATUS_BAD_PARAM,
+ *   ::MLUOP_STATUS_ARCH_MISMATCH, ::MLUOP_STATUS_NOT_SUPPORTED
+ *
+ * @par Data Type
+ * - This function supports the following data types for input tensor \b gates , \b indices ,
+ *   \b locations , \b dispatch and output tensor \b grad_input .
+ *   - gates tensor: half, float.
+ *   - indices tensor: half, float.
+ *   - locations tensor: half, float.
+ *   - dispatch tensor: half, float.
+ *   - grad_input tensor: half, float.
+ *
+ * @par Scale Limitation
+ * - The shape of \b rois should be [boxes_num, 7].
+ * - The shape of \b pts should be [pts_num, 3].
+ * - The shape of \b pts_feature should be [pts_num, channels].
+ * - The shape of \b argmax should be [boxes_num, out_x, out_y, out_z, channels].
+ * - The shape of \b pts_idx_of_voxels should be [boxes_num, out_x, out_y, out_z, max_pts_each_voxel].
+ * - The shape of \b pooled_features should be [boxes_num, out_x, out_y, out_z, channels].
+ *
+ * @par API Dependency
+ * - None.
+ *
+ * @note
```

Comment:
缺少sample字段等，按wiki规范修改
```suggestion
 * @par note
```